webpack/scenes/AnsibleCollections/Details/AnsibleCollectionDetails.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import { Nav, NavItem, TabPane, TabContent, TabContainer, Grid, Row, Col } from 'patternfly-react';
+import BreadcrumbsBar from 'foremanReact/components/BreadcrumbBar';
+import { PropTypes } from 'prop-types';
+import { LoadingState } from '../../../move_to_pf/LoadingState';
+import api from '../../../services/api';
+import AnsibleCollectionDetailsInfo from './AnsibleCollectionDetailsInfo';
+import AnsibleCollectionDetailRepositories from '../../ModuleStreams/Details/Repositories/ModuleStreamDetailRepositories';
```

Comment:
@johnpmitsch : This repositories component would be a complete rewrite for Collections. We could move this out as a utility component?

Reply:
+1